.gitignore

```diff
@@ -1,2 +1,7 @@
-node_modules/*
-src/jspm_packages/*
\ No newline at end of file
+node_modules/
+src/jspm_packages/
+dist/
+
+# TODO: Remove
```

Comment:
Yes. Those files current go into root. I'd prefer them to be in `build/` and `dist/`, but I haven't found a way to do so without breaking everything. Wherever they go, they should not be committed to the repo, but rather be generated in the by `gulp build`. For the time being, they're stuck in root until I find a better solution.